.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,37 @@
+name: Build Docker
+
+on:
+  push:
+    branches:
+      - 'flutter-*'
```

Comment:
Question: Are two different Docker images be generated for branches `flutter-2.2.1` and `flutter-2.2.1-tizen`?

When I create a new branch (say) `flutter-2.4.0`, will the corresponding Docker image be generated automatically? Is there anything I need to know?

Reply:
Yes, the image will be generated and pushed automatically when you push any changes to flutter-* branches. But currently it will have same tag `latest`. I think it is better to push with multiple tags `latest` and the branch name `flutter-2.4.0-tizen` if the container registry allows the quota.

Reply:
Engine 버전을 업데이트하는 동안에는 보통 서로 다른 두 DEPS가 동시에 사용이 됩니다. 예를 들어 제가 flutter-2.4.0-tizen에 첫 commit과 PR을 쓰더라도 기존의 PR들은 flutter-2.2.1-tizen을 타게팅하고 있기 때문에 2.4.0의 DEPS를 사용한 빌드와 2.2.1의 DEPS를 사용한 빌드가 자주 번갈아서 일어나게 됩니다. 빌드 전에 gclient sync를 항상 돌리기 때문에 큰 문제가 없을 수는 있지만 의도한 동작이 맞는지는 확인해보시기 바랍니다.

Reply:
이 Docker 이미지에서 `gclient sync`를 하는 이유는 engine 빌드시 `gclient sync` 시간이 상대적으로 오래걸리는 dart, skia 관련 repository의 .git directory를 이미지내에 미리 cache하기 위함입니다. 말씀하신 상황에서는 cache된 .git 디렉토리에서 commit hash를 찾지 못해 (shallow clone이므로) gclient sync가 조금 더 오래 걸리는 상황이 있긴 하겠지만, 큰 문제가 되지는 않을 것 같습니다.

Reply:
Can we use Korean now? 👀   
BTW, What's wrong with your tone? Who use translator? 😄

Reply:
@bwikbs I rarely use Korean for some comments that I can't express in English quickly, only if the topic is not important for other reviewers.

Reply:
Oh, I see. Now talk about ``translator`` part :smile:

Reply:
What's wrong with my tone 😢

I try to use correct spelling and grammar to help Google translator translate my Korean comments.